src/js/helpers/pageAndSortHelper.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * pageAndSortHelper.js
+ * created by Jonathan Hill 01/07/2020
+ */
+
+export const pageAndSort = (data, sortFunction, page, limit, sortDirection, sortProperty) => {
+    // protects against required props
+    if (!data || !data.length || !page || !limit) return [];
+    // protects against required sorting props
+    if (!sortFunction && !sortDirection && !sortProperty) return [];
+    const defaultSortFunction = (a, b) => {
```

Comment:
Recommend defining this outside of pageAndSort

Reply:
not sure why?

Reply:
it reduces the runtime space complexity of this fn. Instead of defining this everytime the fn is run, you can just define it once.